src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,271 @@
+//                    System.out.println("You have a total of " + i + " tasks in the list.");
+//                }
+//            } else if (str.contains("deadline")) {
+//                String task = sc.nextLine();
+//                store = store + task;
+//                if (task.equals("")) {
+//                    throw new DukeException("The description of a task cannot be empty!");
+//                } else if (!task.contains("/by")) {
+//                    throw new DukeException("Invalid input for a deadline!");
+//                } else {
+//                    Deadline deadline = new Deadline(task);
+//                    try {
+//                        String taskDescription = deadline.getTask();
+//                        storage.add(store);
+//                        list.add(deadline);
+//                        i++;
+//                        System.out.println("I've added this task to the list:");
+//                        System.out.println("  " + taskDescription);
+//                        System.out.println("You have a total of " + i + " tasks in the list.");
+//                    } catch (NullPointerException e) {
+//                        System.out.println("Invalid date input!");
+//                    }
+//                }
+//            } else if (str.contains("event")) {
+//                String task = sc.nextLine();
+//                store = store + task;
+//                if (task.equals("")) {
+//                    throw new DukeException("The description of a task cannot be empty!");
+//                } else if (!task.contains("/at")) {
+//                    throw new DukeException("Invalid input for an event!");
+//                } else {
+//                    Event event = new Event(task);
+//                    try {
+//                        String taskDescription = event.getTask();
+//                        storage.add(store);
+//                        list.add(event);
+//                        i++;
+//                        System.out.println("I've added this task to the list:");
+//                        System.out.println("  " + taskDescription);
+//                        System.out.println("You have a total of " + i + " tasks in the list.");
+//                    } catch (NullPointerException e) {
+//                        System.out.println("Invalid date input!");
+//                    }
+//                }
+//            } else {
+//                throw new DukeException("Invalid input!");
+//            }
+//        }
+//
+//        System.out.println("Thanks for having me!\nHave a nice day ahead!");
+//
+//        try {
+//            BufferedWriter writer = new BufferedWriter(new FileWriter("savedList.txt"));
+//            for (int j = 0; j < storage.size(); j++) {
+//                writer.write(storage.get(j) + "\n");
+//            }
+//            writer.close();
+//        } catch (IOException e) {}
+//    }
+//}
```

Comment:
Perhaps you can remove the long chunk of commented out code if it's not being used!